src/bin/pg_autoctl/monitor.c

```diff
@@ -346,6 +358,266 @@ monitor_node_active(Monitor *monitor,
 }
 
 
+/*
+ * monitor_set_node_candidate_priority updates the monitor on the changes
+ * in the node candidate priority.
+ */
+bool
+monitor_set_node_candidate_priority(Monitor *monitor, int nodeid, char* nodeName, int nodePort, int candidate_priority)
+{
+	PGSQL *pgsql = &monitor->pgsql;
+	const char *sql = "SELECT * FROM pgautofailover.set_node_candidate_priority($1, $2, $3, $4)";
+	int paramCount = 4;
+	Oid paramTypes[4] = { INT4OID, TEXTOID, INT4OID, INT4OID};
+	const char *paramValues[4];
+	char *candidatePriorityText = intToString(candidate_priority).strValue;
+	bool success = true;
+
+	paramValues[0] = intToString(nodeid).strValue;
+	paramValues[1] = nodeName,
+	paramValues[2] = intToString(nodePort).strValue;
+	paramValues[3] = candidatePriorityText;
+
+	if (!pgsql_execute_with_params(pgsql, sql,
+								   paramCount, paramTypes, paramValues,
+								   NULL, NULL))
+	{
+		log_error("Failed to update node candidate priority on node %d"
+				  " for candidate_priority : %s",
```

Comment:
I think we should put the value in double-quotes to make it easier to read the error message here.